frontend/src/__tests__/cypress/cypress/support/e2e.ts

```diff
@@ -325,6 +326,10 @@ beforeEach(function beforeEachHook(this: Mocha.Context) {
   if (Cypress.env('MOCK')) {
     // Fallback: return 404 for all API requests.
     cy.intercept({ pathname: '/api/**' }, { statusCode: 404 });
+
+    // Set up module federation mocks
+    cy.setupModuleFederationMocks(['modelRegistry']);
+
```

Comment:
Disabling all of the module federation routes for now should be sufficient instead of trying to come up with a fancy mock solution.
```suggestion
    // Return 404 for all module federation requests.
    cy.intercept({ pathname: '/_mf/**' }, { statusCode: 404 });
```